sonar-html-plugin/src/test/resources/checks/LayoutTableCheck.html

```diff
@@ -0,0 +1,29 @@
+<table> <!-- Compliant -->
+  <tr>
+    <td></td>
+  </tr>
+</table>
+
+<table role="">
+  <tr>
+    <td></td>
+  </tr>
+</table>
+
+<table role="dummy">
+  <tr>
+    <td></td>
+  </tr>
+</table>
+
+<table role="presentation"> <!-- Non-Compliant -->
+  <tr>
+    <td></td>
+  </tr>
+</table>
+
+<table role="none"> <!-- Non-Compliant -->
+  <tr>
+    <td></td>
+  </tr>
+</table>
```

Comment:
You could add a case with uppercase or mixed letters `role="NONE"`, as well as another with custom tag library prefix to show it does not raise an issue.